virtualization/windowscontainers/quick-start/quick-start-windows-server.md

```diff
@@ -18,19 +18,39 @@ This quick start is specific to Windows Server containers on Windows Server 2016
 
 **Prerequisites:**
 
-One computer system (physical or virtual) running Windows Server 2016. If you are using Windows Server 2016 TP5, please update to [Window Server 2016 Evaluation](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016 ).
+One computer system (physical or virtual) running Windows Server 2016. You can get Windows Server 2016 evaluate license here [Window Server 2016 Evaluation](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016 ).
```

Comment:
If these instructions are meant to cover both installation on both Server 2016 and Server 1709 then both versions should be referenced here.